packages/components/src/index.ts

```diff
@@ -88,19 +132,47 @@ export { default as FormToggle } from './form-toggle';
 export { default as FormTokenField } from './form-token-field';
 export { default as GradientPicker } from './gradient-picker';
 export { default as CustomGradientPicker } from './custom-gradient-picker';
-export { Grid as __experimentalGrid } from './grid';
+export {
+	/**
+	 * @deprecated Import `Grid` instead.
+	 */
+	Grid as __experimentalGrid,
+	Grid,
+} from './grid';
 export { default as Guide } from './guide';
 export { default as GuidePage } from './guide/page';
-export { Heading as __experimentalHeading } from './heading';
-export { HStack as __experimentalHStack } from './h-stack';
+export {
+	/**
+	 * @deprecated Import `Heading` instead.
+	 */
+	Heading as __experimentalHeading,
+	Heading,
+} from './heading';
+export {
+	/**
+	 * @deprecated Import `HStack` instead.
+	 */
+	HStack as __experimentalHStack,
+	HStack,
+} from './h-stack';
 export { default as Icon } from './icon';
 export type { IconType } from './icon';
 export { default as IconButton } from './button/deprecated';
 export {
+	/**
+	 * @deprecated Import `ItemGroup` instead.
+	 */
 	ItemGroup as __experimentalItemGroup,
 	Item as __experimentalItem,
```

Comment:
I forgot to  de`__experimetal`ize the `Item` here, though I wonder if I should even do it for subcomponents as per your comment in https://github.com/WordPress/gutenberg/pull/60837/files/ce4f6386c86c9f4b3a119888aa1ad1326f373ef4..51d389b4cb73811b2eed88aab931abf75f5486ed#r1570925958. Also see: https://github.com/WordPress/gutenberg/pull/60884/files#r1571626137. Any insights appreciated :)